modules/admin_manual/pages/installation/manual_installation/manual_imagick7.adoc

```diff
@@ -4,6 +4,17 @@
 
 ImageMagic shipped for Ubuntu 18.04 or 20.04 is based on version 6, the corresponding `php-imagick` wrapper on version 3.4 which does not have additional capabilities to render patricular image types like HEIC or SVG. To install the latest version with many additional image and video capabilities for use with PHP, you must first uninstall and remove any former version of imagick and the wrapper and install the latest imagick and php-imagick version.
 
+== Backup the ImageMagic Configuration Files
+
+It is recommended to backup existing configuration files from ImageMagic-6. This makes a transition smooth, because changes made can be easier transported for ImageMagic-7. Use the following command to backup them:
```

Comment:
```suggestion
It is recommended that existing configuration files are backed up from ImageMagic-6. This makes a transition smooth, because changes made can be more easily transported for ImageMagic-7. Use the following command to back them up:
```